src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
Do you mean by the trader? The pathfinder uses it by sending it to the trader. The trader uses it to fill out the payment template object.